tests/standalone/write-error-001.phpt

```diff
@@ -17,13 +17,14 @@ $hannes_id = $bulk->insert($hannes);
 $w = 2;
 $wtimeout = 1000;
 $writeConcern = new \MongoDB\Driver\WriteConcern($w, $wtimeout);
-throws(function() use($bulk, $writeConcern, $manager) {
+echo throws(function() use($bulk, $writeConcern, $manager) {
     $result = $manager->executeBulkWrite(NS, $bulk, $writeConcern);
-}, "MongoDB\Driver\Exception\BulkWriteException");
+}, "MongoDB\Driver\Exception\BulkWriteException"),  "\n";
 
 ?>
 ===DONE===
 <?php exit(0); ?>
 --EXPECTF--
 OK: Got MongoDB\Driver\Exception\BulkWriteException
+cannot use 'w' > 1 when a host is not replicated
```

Comment:
This error message dates back to https://github.com/mongodb/mongo/commit/13226efcd6638b707e0f4a64564d655cea9bc9c7#diff-750135e45674cd2fe8dc31115771a990R64 (server 2.6.0) and hasn't changed since, so we should be fine without a pattern here. `EXPECTF` and be changed to `EXPECT`.